zulipterminal/themes/gruvbox_dark.py

```diff
@@ -68,6 +68,7 @@
     'area:help'        : (Color.DARK0_HARD,            Color.BRIGHT_GREEN),
     'area:msg'         : (Color.DARK0_HARD,            Color.NEUTRAL_PURPLE),
     'area:stream'      : (Color.DARK0_HARD,            Color.BRIGHT_BLUE),
+    'area:topic'       : (Color.DARK0_HARD,            Color.BRIGHT_BLUE),
```

Comment:
It is useful to have distinct styles to enable changing them later, but I'm wondering if we should use different colors, rather than simply having duplicates of `area:stream`.